test-configs.yaml

```diff
@@ -1551,6 +1552,7 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - ltp-ipc
```

Comment:
Tested OK with staging-next, although the board often fails to boot for other reasons:
https://staging.kernelci.org/test/plan/id/5fa34d44ed8da013c031e5b4/